target/linux/ramips/image/mt7621.mk

```diff
@@ -235,6 +235,22 @@ define Device/k2p
 endef
 TARGET_DEVICES += k2p
 
+define Device/mir3p
```

Comment:
@iscilyas This should be `Device/xiaomi_mir3p` so you get proper `SUPPORTED_DEVICES := xiaomi,mir3p` automatic variable generated, which is then used for sysupgrade.

Reply:
Thanks for the catch!

I hadn't gotten around to testing sysupgrade (after the previous reviewer's comments). You probably saved me a lot of time.

Fixed.

Reply:
OK just tested sysupgrade. Fixed for real.